doc/devel/triage.rst

```diff
@@ -0,0 +1,214 @@
+
+.. _bug_triaging:
+
+Bug triaging and issue curation
+================================
+
+The `issue tracker <https://github.com/matplotlib/matplotlib/issues>`_
+is important to the communication in the project: it is where users to
+make feature requests and report bugs, developers identify major
+projects to work on and discuss priorities. For this reason, it is
+important to curate the issue list, adding labels to issues and
+closing issues that are not necessary.
+
+
+Triaging issues does not require any particular expertise in the
+internals of Matplotlib, is extremely valuable to the project, and we
+welcome anyone to participate in issue triage!  However, people who
+are not part of the Matplotlib organization do not have `permissions
+to change milestones, add labels, or close issue
+<https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization>`_.
+If you do not have enough permissions do to a step, please leave a
+comment tagging ``@matplotlib/triage_team`` with your recommendations.
+
+Working on issues to improve them
+---------------------------------
+
+Improving issues increases their chances of being successfully resolved.
+Guidelines on submitting good issues can be found :ref:`here
+<submitting-a-bug-report>`.
+A third party can give useful feedback or even add
+comments on the issue.
+The following actions are typically useful:
+
+  - documenting issues that are missing elements to reproduce the problem
+    such as code samples
+
+  - suggesting better use of code formatting
+
+  - suggesting to reformulate the title and description to make them more
+    explicit about the problem to be solved
+
+  - linking to related issues or discussions while briefly describing
+    how they are related, for instance "See also #xyz for a similar
+    attempt at this" or "See also #xyz where the same thing was
+    reported" provides context and helps the discussion
+
+  - verifying that the issue is reproducible
+
+  - classify the issue as a feature request, a long standing bug or a
+    regression
+
+.. topic:: Fruitful discussions
+
+   Online discussions may be harder than it seems at first glance, in
+   particular given that a person new to open-source may have a very
+   different understanding of the process than a seasonned maintainer.
```

Comment:
```suggestion
   different understanding of the process than a seasoned maintainer.
```

Reply:
This was un-edited from the sklean text, going to open a PR with them too :-p